style.css

```diff
@@ -0,0 +1,2496 @@
+}
+
+/*Puzzle 2*/
+#image0 {
+    position: relative;
+    width: 116%;
+    height: 115%;
+    z-index: 2;
+}
+#image1 {
+    position: relative;
+    width: 116%;
+    height: 115%;
+    left: -4%;
+    z-index: 2;
+}
+#image2{
+    position: relative;
+    width: 116%;
+    height: 115%;
+    left: -16%;
+    z-index: 2;
+}
+#image3{
+    position: relative;
+    width: 116%;
+    height: 110%;
+    bottom: 2%;
+    z-index: 2;
+}
+#image4{
+    position: relative;
+    width: 110%;
+    height: 110%;
+    bottom: 2.4%;
+    left: -4%;
+    z-index: 2;
+}
+#image5{
+    position: relative;
+    width: 116%;
+    height: 110%;
+    bottom: 2%;
+    left: -16%;
+    z-index: 2;
+}
+#image6{
+    position: relative;
+    width: 116%;
+    height: 110%;
+    bottom: 10%;
+    z-index: 2;
+}
+#image7{
+    position: relative;
+    width: 110%;
+    height: 109%;
+    bottom: 9%;
+    left: -3%;
+    z-index: 2;
+}
+#image8{
+    position: relative;
+    width: 116%;
+    height: 110%;
+    left: -16%;
+    bottom: 10%;
+    z-index: 2;
+}
+
+/*Puzzle 3*/
+#images0{
+    position: relative;
+    width: 105%;
+    height: 109%;
+    z-index: 2;
+}
+#images1{
+    position: relative;
+    width: 118%;
+    height: 110%;
+    left: -15%;
+    z-index: 2;
+}
+#images2{
+    position: relative;
+    width: 129%;
+    height: 110%;
+    left: -15%;
+    z-index: 2;
+}
+#images3{
+    position: relative;
+    width: 104%;
+    height: 125%;
+    left: -4%;
+    z-index: 2;
+}
+#images4{
+    position: relative;
+    width: 121%;
+    height: 137%;
+    z-index: 2;
+    bottom: 12%;
+}
+#images5{
+    position: relative;
+    width: 100%;
+    height: 120%;
+    z-index: 2;
+    bottom: 11%;
+    left: 1%;
+}
+#images6{
+    position: relative;
+    width: 135%;
+    height: 137%;
+    left: -21%;
+    bottom: 11%;
+    z-index: 2;
+}
+#images7{
+    position: relative;
+    width: 105%;
+    height: 100%;
+    z-index: 2;
+    bottom: -6%;
+    left: -5%;
+}
+#images8{
+    position: relative;
+    width: 125%;
+    height: 112%;
+    bottom: -7%;
+    z-index: 2;
+}
+#images9{
+    position: relative;
+    width: 115%;
+    height: 110%;
+    bottom: 9%;
+    left: 1%;
+    z-index: 2;
+}
+#images10{
+    position: relative;
+    width: 119%;
+    height: 96%;
+    left: -5%;
+    bottom: -6%;
+    z-index: 2;
+}
+#images11{
+    position: relative;
+    width: 107%;
+    height: 125%;
+    left: -7%;
+    bottom: 10%;
+    z-index: 2;
+}
+#images12{
+    position: relative;
+    width: 105%;
+    height: 100%;
+    z-index: 2;
+}
+#images13{
+    position: relative;
+    width: 130%;
+    height: 118%;
+    bottom: 18%;
+    left: -15%;
+    z-index: 2;
+}
+#images14{
+    position: relative;
+    width: 120%;
+    height: 118%;
+    z-index: 2;
+    bottom: 18%;
+    left: -5%;
+}
+#images15{
+    position: relative;
+    width: 105%;
+    height: 100%;
+    z-index: 2;
+    left: -5%;
+}
+
+}
+
+@media (max-width: 767px) and (orientation: landscape){
```

Comment:
Great that you are considering portrait vs landscape! Helps with the responsiveness for sure.